modules/ocf/manifests/staff_users.pp

```diff
@@ -1,11 +1,10 @@
 # Create home directories for staff members if NFS isn't mounted.
 
-class ocf::staff_users {
+class ocf::staff_users($noop = false) {
   # Only create home directories if:
   #   - we don't mount NFS
-  #   - we're a server (not a desktop)
-  #   - we're not the file server
-  if !str2bool($::ocf_nfs) and $::type == 'server' and !tagged('ocf_filehost') {
+  #   - it hasn't been turned off in Hiera
+  if !str2bool($::ocf_nfs) and !$noop {
```

Comment:
i think this entire design is bad; we should instead just add pam_mkhomedir which fixes this